config/initializers/elasticsearch.rb

```diff
@@ -0,0 +1,11 @@
+require 'typhoeus/adapters/faraday'
```

Comment:
I'm confused about the relationships between Elasticsearch, Estella, Typhoeus and Faraday. Why did we need to add Typhoeus to the Gemfile just to use the Faraday adapter?